client/src/components/App.jsx

```diff
@@ -17,11 +17,12 @@ class App extends React.Component {
             modalIsOpen: false,
             modal: '',
             hover: false,
-            currStart: 0,
+            currStart: 2,
             arrows: false
         } 
         
-        // this.getGalleryData = this.getGalleryData.bind(this);
+        this.getGalleryData = this.getGalleryData.bind(this);
+        this.addIndex = this.addIndex.bind(this);
         this.openModal = this.openModal.bind(this);
         // this.afterOpenModal = this.afterOpenModal.bind(this);
```

Comment:
Will this be used later?

Reply:
Probably not, just leaving it in as a model. Since I'm nearly done with my modal, I probably won't use it.